locales/ja.json

```
@@ -1,823 +1,938 @@
+      "pass": "ドキュメントに同じid属性を共有する静的な要素はありません",
+      "fail": "ドキュメントに同じid属性を共有する静的な要素が複数存在します"
+    },
+    "aria-label": {
+      "pass": "aria-label属性が存在し、空ではありません",
+      "fail": "aria-label属性が存在しない、または空です"
+    },
+    "aria-labelledby": {
+      "pass": "aria-labelledby属性が存在し、スクリーン・リーダーに認識可能な要素を参照しています",
+      "fail": "aria-labelledby属性が存在しない、存在しない要素を参照している、または空の要素を参照しています",
+      "incomplete": "aria-labelledby属性が、存在する要素を参照していることを確認しましょう"
+    },
+    "avoid-inline-spacing": {
+      "pass": "テキストの間隔に影響する '!important' のついたインラインスタイルは指定されていません",
+      "fail": {
+        "singular": "ほとんどのブラウザーで上書きすることはサポートされていないため、インラインスタイル ${data.values} から '!important' を削除します",
+        "plural": "ほとんどのブラウザーで上書きすることはサポートされていないため、インラインスタイル ${data.values} から '!important' を削除します"
+      }
+    },
+    "button-has-visible-text": {
+      "pass": "要素にスクリーン・リーダーが認識可能な内部テキストが存在しています",
+      "fail": "要素にスクリーン・リーダーが認識可能な内部テキストが存在していません",
+      "incomplete": "要素に子ノードがあるか判定できません"
+    },
+    "doc-has-title": {
+      "pass": "ドキュメントに空ではない<title>要素が存在しています",
+      "fail": "ドキュメントに空ではない<title>要素が存在していません"
+    },
+    "exists": {
+      "pass": "要素は存在していません",
+      "incomplete": "要素が存在しています"
+    },
+    "has-alt": {
+      "pass": "要素にalt属性が存在しています",
+      "fail": "要素にalt属性が存在していません"
+    },
+    "has-visible-text": {
+      "pass": "要素にスクリーン・リーダーが認識可能なテキストが存在しています",
+      "fail": "要素にスクリーン・リーダーが認識可能なテキストが存在していません",
+      "incomplete": "要素に子ノードがあるか判定できません"
+    },
+    "is-on-screen": {
+      "pass": "要素が表示されていません",
+      "fail": "要素が表示されています"
+    },
+    "non-empty-alt": {
+      "pass": "要素に空ではないalt属性が存在しています",
+      "fail": {
+        "noAttr": "要素にalt属性が指定されていません",
+        "emptyAttr": "要素に空のalt属性が指定されています"
+      }
+    },
+    "non-empty-if-present": {
+      "pass": {
+        "default": "要素に value 属性が存在しません",
+        "has-label": "要素に空ではない value 属性が存在しています"
+      },
+      "fail": "要素にvalue属性が存在し、value属性が空です"
+    },
+    "non-empty-placeholder": {
+      "pass": "要素にはplaceholder属性が指定されています",
+      "fail": {
+        "noAttr": "要素にはplacehoder属性が指定されていません",
+        "emptyAttr": "要素には空のplacehoder属性が指定されています"
+      }
+    },
+    "non-empty-title": {
+      "pass": "要素にtitle属性が存在しています",
+      "fail": {
+        "noAttr": "要素にはtitle属性が指定されていません",
+        "emptyAttr": "要素には空のtitle属性が指定されています"
+      }
+    },
+    "non-empty-value": {
+      "pass": "要素に空ではないvalue属性が存在しています",
+      "fail": {
+        "noAttr": "要素にはvalue属性が指定されていません",
+        "emptyAttr": "要素には空のvalue属性が指定されています"
+      }
+    },
+    "presentational-role": {
+      "pass": "要素のデフォルトのセマンティクスはrole=\"${data.role}\"で上書きされました",
+      "fail": {
+        "default": "要素のデフォルトのセマンティクスは role=\"none\" または role=\"presentation\"で上書きされませんでした",
+        "globalAria": "要素にはARIAのグローバル属性が指定されているため、roleはpresentation系のものになりません",
+        "focusable": "要素がフォーカス可能なため、roleはpresentation系のものにはなりません",
+        "both": "ようそにはARIAのグローバル属性が指定されており、フォーカス可能なため、roleはpresentation系のものにはなりません"
```

```suggestion
        "globalAria": "要素にはARIAのグローバル属性が指定されているため、roleはpresentation系のものになりません",
        "focusable": "要素がフォーカス可能なため、roleはpresentation系のものになりません",
        "both": "要素にARIAのグローバル属性が指定されておりフォーカス可能なため、roleはpresentation系のものになりません"
```
「要素」の変換漏れをひとつ修正しました。
「presentation系のもの」という訳は修正していないのですが少し違和感がありました。既存の訳ではないと思うので、少し検討させてください。

---

初めて見ると伝わりづらい訳ですね。
ただチェックの内容を考えると他にあまり良い案もなく。
意訳して、 "none" または "presentation" ではない、とするのも1つの方法かもしれません。

---

```suggestion
        "globalAria": "要素にはARIAのグローバル属性が指定されているため、role=\"none\"またはrole=\"presentation\"にはなりません",
        "focusable": "要素がフォーカス可能なため、role=\"none\"またはrole=\"presentation\"にはなりません",
        "both": "要素にはARIAのグローバル属性が指定されており、フォーカス可能なため、role=\"none\"またはrole=\"presentation\"にはなりません"
```
確認ありがとうございます。そうですね、ということで、修正案を修正しましたので再度ご確認いただけますでしょうか。